docs/notebooks/inequality_constraints.pct.py

```diff
@@ -148,16 +153,10 @@ def create_bo_model(data):
 
 # %%
 constraint_data = result.datasets[CONSTRAINT]
-new_data = (
-    constraint_data.query_points[-num_steps:], constraint_data.observations[-num_steps:]
-)
+new_data = (constraint_data.query_points[-num_steps:], constraint_data.observations[-num_steps:])
```

Comment:
Is this another black format? I intentionally split lines like this to avoid a horizontal scroll bar on the docs site

Reply:
Therefore, you have to use `,` after the last entry. `black` would ignore ignore it.
Initial thing was:

```
new_data = (
      constraint_data.query_points[-num_steps:], constraint_data.observations[-num_steps:]
)
```

Actually, I don't think that neither of them are good.